bundles/org.openhab.voice.pollytts/README.md

```diff
@@ -43,6 +43,40 @@ A value of 365 removes files that have been unused for a year.
 Use "default" to select the system default audio format.
 The default audio format can be overriden with the value "mp3" or "ogg".
 
+
+### Service Configuration via Text files
+
+PollyTTS can also be installed and configured textually:
+
+#### Installing
+
+In your `$OPENHAB_ROOT/conf/services/addons.cfg` add:
+
+```
+...other stuff...
+
+voice = pollytts
+
+...other stuff...
+```
+
+This will install the Polly TTS voice addon. 
```

Comment:
Please remove theselines. generic addon installation instructions do not belong into the addon documentation.